src/components/common/multi-entry/multi-entry.tsx

```diff
@@ -26,15 +24,15 @@ export const MultiEntry = ({
         <MultiEntryAccordionItem title={title}>
             <>
                 {children}
-                <AddMultiEntryButton
+                <ButtonWithIcon.Default
```

Comment:
- button needs 100% width
- button shadow is getting cropped off by accordion, it used to have 2rem margin and that prevented the cropping

Reply:
thanks for the change but the update causes it not to match the Figma mocks, I think you need to have
- 1rem top and bottom margins for the button
- reduce the accordion item bottom margin to 1rem